### PR TITLE
Resolve issue for Añonuevo, Jheferson

### LINKED ISSUE #5: Code Review for Anonuevo
### Required
- **Clean Repository Organization**
  - [x] Consolidate assets into a common folder to avoid clutter in the project directory.
  image
       [View Guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR1pphsWlliw0L_vbPJKbRCpW81Gt151vhOUq4iEw0Ftms7GjjedbKMX-fI_aem_AaHDPId1LpnIKTLO8jtJYQQGefS9ZBVU9dr1OCxS88dCO9DgcNB0UTMObNs4Ok6LM47wdHt4CyLzGnWZQJ0fiCb9#slide=id.g26e9a7bd30a_1_0)
   - [x] All files should be named in snake case, all lowercase. Please rename your `ex9-tuf_developers/anonuevo_jheferson/jhefstyle.css` file to `jhef_style.css` or choose a more descriptive and straightforward filename like `style.css`.
          [View Guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR1pphsWlliw0L_vbPJKbRCpW81Gt151vhOUq4iEw0Ftms7GjjedbKMX-fI_aem_AaHDPId1LpnIKTLO8jtJYQQGefS9ZBVU9dr1OCxS88dCO9DgcNB0UTMObNs4Ok6LM47wdHt4CyLzGnWZQJ0fiCb9#slide=id.g25d18931cb7_3_0)

- **Readability in CSS Selectors**
  - [x] Separate CSS blocks with a single space 
           - also add a single space after selectors and before "{" which is not implemented on your current code.
  https://github.com/PUP-BSIT/ex9-tuf_developers/blob/1f0add269054924241347e881d787ec0abc053a9/anonuevo_jheferson/jhefstyle.css#L110-L117 [View Guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24acb740da7_0_35)

- **Remove Unnecessary Comments**
  - [x] Eliminate redundant or unnecessary comments.
  https://github.com/PUP-BSIT/ex9-tuf_developers/blob/1f0add269054924241347e881d787ec0abc053a9/anonuevo_jheferson/jhefstyle.css#L109 [View Guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5)

- **Proper Indentation**
  - [x] Ensure consistent and proper indentation.
  https://github.com/PUP-BSIT/ex9-tuf_developers/blob/1f0add269054924241347e881d787ec0abc053a9/anonuevo_jheferson/jhefstyle.css#L119-L160 [View Guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24acb740da7_0_50)

### PR DESCRIPTION
**Pull request by Anonuevo-Jheferson**
- [x] Consolidate assets into a common folder - 9953d994da1d44322430d2eaaee6fd248c6e6ad4
- [x] All files should be named in snake case, all lowercase. - 77bdd1038b0d55c5f2cab3e4dfe27a3f8b16b07d
- [x] Separate CSS blocks with a single space - 46411b58ec756741337884f7fdfc56fc2e579cbe
- [x] Eliminate redundant or unnecessary comments. - d7728e3c911b13d83471beecfc46040fff43be54
- [x] Ensure consistent and proper indentation. - 3ea51d40861ecff1dcdd01d751801b0c51f22c25

### CODE REVIEW

File: anonuevo_jheferson/index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            id="email" />
+          </div>
+          <div>
+            <label class="form-label" for="fullname">Fullname</label>
+            <input
+              class="form-input"
+              type="text"
+              name="fullname"
+              id="fullname"
+            />
```
Comment (go-christianharrel) [inline anonuevo_jheferson/index.html]:
```suggestion
```

File: anonuevo_jheferson/index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <input class="form-input" type="email" name="email" 
+            id="email" />
+          </div>
+          <div>
+            <label class="form-label" for="fullname">Fullname</label>
+            <input
+              class="form-input"
+              type="text"
+              name="fullname"
+              id="fullname"
```
Comment (go-christianharrel) [inline anonuevo_jheferson/index.html]:
```suggestion
              id="fullname" />
```

File: anonuevo_jheferson/index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            />
+          </div>
+          <div>
+            <label class="form-label" for="age">Age</label>
+            <input class="form-input" type="number" name="age" id="age" />
+          </div>
+          <div>
+            <label class="form-label" for="subject">Subject</label>
+            <input class="form-input" type="text" name="subject" 
+            id="subject" />
```
Comment (go-christianharrel) [inline anonuevo_jheferson/index.html]:
```suggestion
              id="subject" />
```

File: anonuevo_jheferson/index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    <div class="title">
+      <h2>Contact Me</h2>
+    </div>
+    <div class="form-container">
+      <form>
+        <div class="form-row">
+          <div>
+            <label class="form-label" for="email">Email</label>
+            <input class="form-input" type="email" name="email" 
+            id="email" />
```
Comment (go-christianharrel) [inline anonuevo_jheferson/index.html]:
```suggestion
              id="email" />
```

File: anonuevo_jheferson/index.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+          <div>
+            <label class="form-label" for="subject">Subject</label>
+            <input class="form-input" type="text" name="subject" 
+            id="subject" />
+          </div>
+        </div>
+        <div class="message-container">
+          <label class="form-label" for="message">Message</label>
+          <textarea class="form-text-area" name="message" cols="30" 
+          rows="10">
```
Comment (go-christianharrel) [inline anonuevo_jheferson/index.html]:
```suggestion
            rows="10">
```

**Review by go-christianharrel — CHANGES_REQUESTED** [review]
Kindly remove [ .vscode folder](https://github.com/PUP-BSIT/ex9-tuf_developers/tree/jheferson-branch/.vscode) in your workspace.

**Comment by Anonuevo-Jheferson** [comment]
> Kindly remove [ .vscode folder](https://github.com/PUP-BSIT/ex9-tuf_developers/tree/jheferson-branch/.vscode) in your workspace.

- [x] remove .vscode folder - f3f1314a428ce35c2cb2484214490813ceb09ac9

**Comment by Anonuevo-Jheferson** [comment]
@go-christianharrel 
- [x] apply all pull request suggestions - 85538a1227f8ac32ffedfd76afa676bf27d3831f

**Review by go-christianharrel — APPROVED** [review]
**LGTM!**

**Pull request merged**